coreneuron/nrnoc/fadvance_core.cpp

```diff
@@ -229,6 +230,44 @@ void nrn_ba(NrnThread* nt, int bat) {
     }
 }
 
+void nrncore2nrn_send_values(NrnThread* nth) {
+  if (nrn2core_trajectory_values_ == nullptr) {
+      // standalone execution : no callbacks
+      return;
+  }
+
+  if (nth == NULL) {
```

Comment:
@nrnhines : if I understood correctly we are using `nth==NULL` as special case here. Wouldn't to better to turn that into separate function?

Intuitively, if `nt==NULL`, we typically don't do anything. May be you can add this block to separate function like `initialize_trajectories`?

Reply:
This was only needed on per step value transfer to tickle the gui in NEURON to initialize
in preparation to plot first points on next call.  There is also the claim for whole trajectory transfer
mode that one needs to initialize vector sizes.  The fragment could easily be put into a separate
function in coreneuron. However it is tedious to declare new callbacks into NEURON so it
seems simplest not to introduce one in the separate function.